web/src/page_params.ts

```diff
@@ -1,7 +1,17 @@
 import $ from "jquery";
 
+type ContributorProfile = {
+    avatar: string;
+    email: string;
+    github_username?: string;
+    name: string;
+};
```

Comment:
I think you can refer to https://github.com/zulip/zulip/blob/main/tools/fetch-contributor-data#L47-L52 for this one.